tests/scale_test/utils.py

```diff
@@ -0,0 +1,238 @@
+import json
+import uuid
+import requests
+import click
+
+
+def send_notification(lira_url, auth_token, notification):
+    """Send a notification to a given Lira.
+
+    Args:
+        lira_url (str): A typical Lira url, e.g. https://pipelines.dev.data.humancellatlas.org/
+        auth_token (str): The Lira auth_token.
+        notification (dict): A dict of notification content.
+
+    Returns:
+        requests.Response: The response object returned by Lira.
+    """
+    response = requests.post(url=lira_url + 'notifications', json=notification, params={'auth': auth_token})
+    return response
+
+
+def prepare_notification(bundle_uuid, bundle_version, subscription_id, label=None, transaction_id=None):
+    """Compose the notification content from given values.
+
+    Args:
+        bundle_uuid (str): A valid Blue Box bundle uuid.
+        bundle_version (str): A valid Blue Box bundle version.
+        subscription_id (str): A valid Lira subscription id in Blue Box.
+        label (str): A label to be added to the notification, which will then be added to the workflow started by Lira.
+        transaction_id (str): A valid transcription id.
+
+    Returns:
+        dict: A dict of valid notification content.
+
+    """
+    notification = {
+        'match': {
+            'bundle_uuid': bundle_uuid,
+            'bundle_version': bundle_version
+        },
+        'subscription_id': subscription_id,
+        'transaction_id': transaction_id or _prepare_transaction_id(),
+        'es_query': {
```

Comment:
Could you move this query to a separate file so that this function is easier to read? Also, could you make the query file a parameter to the function to make it more generalized?

Reply:
Yes, it would be better! The only issue with it is this qeury will be duplicated with thoses quries in the Lira repo. But it's OK to separate it for now. Will do.